YACS-RCOS/yacs.n
# Performance - Handle duplicate /class request using Vuex course list

**Issue**
closes #325 

**Database Changes/Migrations**
N/A

**Test Modifications**
N/A

**Test Procedure**
1. open yacs
2. Click course explorer
3. Notice way faster load time!
4. Click any major
5. Notice way faster load time!
6. Click any course
7. Notice way faster load time!

**Photos**
N/A

**Additional Info**
Changes mainly consisted of adding the vuex state and then moving instance variables of pages to computed functions
